packages/test-base/src/androidAndroidTest/kotlin/io/realm/kotlin/test/shared/RealmTests.kt

```diff
@@ -658,6 +659,123 @@ class RealmTests {
         }
     }
 
+    @Test
+    fun assetFile() {
+        val config = RealmConfiguration.Builder(setOf(Parent::class, Child::class))
+            .directory(tmpDir)
+            // Need a separate name to avoid clashes with this.realm initialized in setup()
+            .name("prefilled.realm")
+            .assetFile("asset.realm")
```

Comment:
I think that it might be worth to have some tests with assets in subdirectories.

Reply:
Yep. Good catch. I have added some tests that uses subdirs